pyproject.toml

```diff
@@ -1,16 +1,16 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "scikit_build_core>=0.8",
     "pybind11>=2.10.0",
     "wheel",
 ]
-build-backend = "setuptools.build_meta"
+build-backend = "scikit_build_core.build"
 
 [project]
 name = "pybtls"
-version = "0.3.7"  # Should also manually update in __init__.py
+version = "0.3.7"
 description = "A package for short-to-mid span bridge traffic loading simulation"
-readme = "README.md"
+readme = "README.rst"
```

Comment:
I detest rst! Why not MD?

Reply:
Hmmmm I just followed the pystra to make the pybtls readme...